resources/core/charts/event-bus/README.md

```diff
@@ -20,6 +20,7 @@ Configure these options for each business requirement:
 | | `global.persistence.size` | The size of the storage volume. |
 | | `global.persistence.maxAge`| The maximum period of time for storing an event (`0` for unlimited). |
 | |`global.natsStreaming.resources`| Refer to Kubernetes resource requests and limits for details. |
+| |`global.natsStreaming.channel.maxInactivity`| Max inactivity (no new event, no subscription) after which a channel can be garbage collected (`0` for unlimited) |
```

Comment:
`The maximum inactivity period (without any new event or subscription) after which a channel can be garbage collected (`0` for unlimited).`